src/equations/compressible_euler_2d.jl

```diff
@@ -1002,11 +1066,57 @@ end
     return SVector(f1m, f2m, f3m, f4m)
 end
 
+@inline function splitting_vanleer_haenel(u, ::Val{:plus},
```

Comment:
Do you have a reference for the curved version?

Reply:
I can add one or two. This curvilinear version comes from several places. I think the simplified energy splitting due to Hänel is the reason it works directly (and why the Drikakis and Tsangris splitting mimics it.

Reply:
That would be great. Thanks!

Reply:
Done